ils_middleware/dags/stanford.py

```diff
@@ -210,12 +210,16 @@ def dag_failure_callback(ctx_dict) -> None:
         task_id="sinopia_update_success_notification",
```

Comment:
Maybe change this task_id to `sinopia_update_notification` since we no longer have separate email success and failure tasks?